File: cadwiki-nuget/NetUtils/Enums.vb
Option Strict On
Option Infer Off
Option Explicit On

Public Class Enums
    Public Shared Function GetStringsFromEnum(type As Type) As List(Of String)
        Dim strings As New List(Of String)
        Dim items As Array = System.Enum.GetNames(type)
        For Each item As String In items
            strings.Add(item)
        Next
        Return strings
    End Function
End Class

File: cadwiki-nuget/cadwiki.NetUtils/Enums.cs
using System;
using System.Collections.Generic;

namespace cadwiki.NetUtils
{

    public class Enums
    {
        public static List<string> GetStringsFromEnum(Type @type)
        {
            var strings = new List<string>();
            Array items = Enum.GetNames(type);
            foreach (string item in items)
                strings.Add(item);
            return strings;
        }
    }
}